marlyna93-Szeged.md

```diff
@@ -0,0 +1,8 @@
+# Szeged, HU
+
+### Restaurants
+
+### Tourist Attractions
+
+### Local Attractions
+- Szeged Symphony Orchestra
```

Comment:
Does the orchestra play year round? Or only during a particular month or season? 🎹 🎸